pkg/network/ebpf/c/protocols/postgres/decoding.h

```diff
@@ -210,11 +234,7 @@ static __always_inline bool handle_response(pktbuf_t pkt, conn_tuple_t conn_tupl
         return 0;
     }
 
-    if (iteration_value->iteration >= POSTGRES_MAX_TAIL_CALLS_FOR_MAX_MESSAGES) {
-        return 0;
-    }
```

Comment:
why did you remove this check?

Reply:
Because I do not count iterations, I count total processed messages

Reply:
I'm not following why you removed the check, it is still a valid check, you might just need to modify the condition